subroutines.sh

```diff
@@ -164,9 +164,16 @@ listen_to_ble() {
 
       MQTT_TOPIC="tesla_ble/$VIN/binary_sensor/presence"
 
+      log_debug "BLE_LN:$BLE_LN"
+      log_debug "BLE_MAC:$BLE_MAC"
+      log_debug "PRESENCE_EXPIRE_TIME:$PRESENCE_EXPIRE_TIME"
+      log_debug "VIN:$VIN"
+      log_debug "macAddr:$macAddr"
+
       # If something matches or BLE_MAC is default
       if echo "$BLTCTL_OUT" | grep -Eq "($BLE_MAC|$BLE_LN)" ||
-        [ $BLE_MAC == "FF:FF:FF:FF:FF:FF" ]; then
+        [ $BLE_MAC == "FF:FF:FF:FF:FF:FF" ] ||
+        [ $PRESENCE_EXPIRE_TIME -eq 0 ]; then
```

Comment:
This is the fix.

At start PRESENCE_EXPIRE_TIME = 0, force call to check_presence.